encoding/base64url.ts

```diff
@@ -17,6 +17,10 @@ export function addPaddingToBase64url(base64url: string): string {
 }
 
 function convertBase64urlToBase64(b64url: string): string {
+  if (!/^[-_A-Z0-9]*?={0,2}$/i.test(b64url)) {
+    // Contains characters not part of base64url spec.
+    throw new TypeError("Invalid base64url character in string!");
```

Comment:
Maybe we should align the error message with `base64.decode` like:

```ts
throw new TypeError("Failed to decode base64url: invalid character);
```

---
BTW @lucacasonato do you think the error here should be DOMException?

Reply:
Changed, for the time being. Awaiting input on whether or not this should be a `DOMException`.